.github/workflows/codeql-analysis.yml

```diff
@@ -17,8 +17,6 @@ jobs:
   analyze:
     if: github.repository == 'cilium/cilium'
     runs-on: ubuntu-18.04
```

Comment:
I assume the line removed was because of the validation?

Reply:
It was, it isn't really enforced by gh, but is used only when `strategy.matrix` is set.